gitpod-io/gitpod
# Update VS Code Browser to `1.75.1`

## Description
Update code to `1.75.1`

## Progress


## How to test

- Switch to VS Code Browser Insiders in settings.
- Start a workspace.
- Test the following:
  - that user data is synced across workspaces as well as on workspace restarts, especially for extensions

## Release Notes

```release-note
NONE
```

## Werft options:
